README.md

```diff
@@ -2,38 +2,285 @@
+   cat checksums.txt | grep notation-azure-kv_${version}_darwin_amd64.tar.gz | shasum -a 256 -c
 
+   # install the plugin
+   mkdir -p "$HOME/Library/Application Support/notation/plugins/azure-kv"
+   tar zxf notation-azure-kv_${version}_darwin_amd64.tar.gz -C "$HOME/Library/Application Support/notation/plugins/azure-kv" notation-azure-kv
+   ```
+   For Windows Powershell:
+   ```powershell
+   $version = "0.6.0"
 
+   # validate checksum
+   (Get-FileHash .\notation-azure-kv_${version}_windows_amd64.zip).Hash
 
+   # install the plugin
+   mkdir "$env:AppData\notation\plugins\azure-kv"
+   Expand-Archive -Path notation-azure-kv_${version}_windows_amd64.zip -DestinationPath "$env:AppData\notation\plugins\azure-kv"
+   ```
+3. Try to run `notation plugin list` to show the installed plugin.
+
+## Getting started with a self-signed Azure Key Vault Certificate
+> **Note** It is suggested to get a certificate from a trusted CA since a self-signed certificate is not publicly trusted.
+1. Install Azure CLI by following the [guide](https://learn.microsoft.com/en-us/cli/azure/install-azure-cli)
+2. Login to Azure with Azure CLI, set the subscription and make sure the `GetCertificate` and `Sign` permission have been granted to your role:
+   ```sh
+   az login
+   az account set --subscription $subscriptionID
+   ```
+3. Create an Azure Key Vault and a self-signed certificate:
+   ```sh
+   resourceGroup=notationResource
+   keyVault=notationKV
+   location=westus
+   certName=notationSelfSignedCert
+
+   # create a resource group
+   az group create -n $resourceGroup -l $location
+   
+   # create a Azure Key Vault
+   az keyvault create -l $location -n $keyVault --resource-group $resourceGroup
+
+   # generate certificate policy
+   cat <<EOF > ./selfSignedPolicy.json
+   {
+     "issuerParameters": {
+       "certificateTransparency": null,
+       "name": "Self"
+     },
+     "keyProperties": {
+       "curve": null,
+       "exportable": false,
+       "keySize": 2048,
+       "keyType": "RSA",
+       "reuseKey": true
+     },
+     "secretProperties": {
+       "contentType": "application/x-pem-file"
+     },
+     "x509CertificateProperties": {
+        "ekus": [
+           "1.3.6.1.5.5.7.3.3"
+       ],
+       "keyUsage": [
+         "digitalSignature"
+       ],
+       "subject": "CN=Test Signer",
+       "validityInMonths": 12
+     }
+   }
+   EOF
+
+   # create self-signed certificate
+   az keyvault certificate create -n $certName --vault-name $keyVault -p @selfSignedPolicy.json
+
+   # get the key identifier
+   keyID=$(az keyvault certificate show -n $certName --vault-name $keyVault --query 'kid' -o tsv)
+   ```
+4. Prepare a container registry. You can follow the [guide](https://learn.microsoft.com/azure/container-registry/container-registry-get-started-portal?tabs=azure-cli) to create an Azure Container Registry which is recommended. Suppose the logging server is `notation.azurecr.io` with an `hello-world:v1` image.
```

Comment:
```suggestion
4. [Create an Azure Container Registry](https://learn.microsoft.com/azure/container-registry/container-registry-get-started-portal?tabs=azure-cli) for storing your signed images. The remaining steps use the example login server `notation.azurecr.io`, but you must substitute your own login server value.
```

Reply:
Also, you need to add a step giving them a sample image to sign and how to pull it into ACR. If the user is creating an ACR from scratch, they likely don't have an image their ACR to sign.

Reply:
Updated.